site_root/input/resources/profiles/StructureDefinition-kfdrc-patient-no-phi.json

```diff
@@ -1,16 +1,16 @@
 {
```

Comment:
Not sure where to put this comment but its similar to the one made on `kfdrc-specimen.json`. I don't think you need the bindings to the `us-core-race` and `us-core-ethnicity` extensions in this profile since they are included in the profile which this one extends.

Reply:
Updated and Validated!